src/gmm_sup.erl

```diff
@@ -53,5 +50,9 @@ init([]) ->
         id => << "inbox_dispatcher" >>,
         start => {inbox, start_link, []}
     },
-    ChildSpecs = [InboxDispatcherSpec | [RedisSpec | outbox:specs_for_supervisor()]],
+    CsvWriterSpecs = #{
+        id => <<"csv_writer">>,
+        start => {instrumentation, start_link, []}
+    },
+    ChildSpecs = [CsvWriterSpecs | [InboxDispatcherSpec | [RedisSpec | outbox:specs_for_supervisor()]]],
```

Comment:
Nie dodawaj się do tej zbrodni na ludzkości, da się to załatwić z mniejszą ilością nawiasów
```suggestion
    ChildSpecs = [CsvWriterSpecs, InboxDispatcherSpec, RedisSpec | outbox:specs_for_supervisor()],
```
Albo operatorem `++`

Reply:
Danke schon